src/Kernel/client/plotting.ts

```diff
@@ -1,12 +1,202 @@
+                        return (amps[idx].Imag);
+                    }),
+                    backgroundColor: "#48bfe3",
+                    borderColor: "#48bfe3",
+                    label: "Imag"
+                }
+            ],
+        },
+        options: {
+            responsive: false,
+            legend: {
+                display: true,
+            },
+            scales: {
+                xAxes: [{
+                    scaleLabel: {
+                        display: true,
+                        labelString: 'Basis States'
+                    },
+                    ticks: {
+                        maxRotation: 0,
+                        minRotation: 0
+                    }
+                }],
+                yAxes: [{
+                    scaleLabel: {
+                        display: true,
+                        labelString: 'Meas. Probability'
+                    },
+                    ticks: {
+                        suggestedMax: 1,
+                        suggestedMin: -1
+                    }
+                }]
+            }
+        }
+    });
+
+};
+
+export function createBarChartAmplitudePhaseOption(element: HTMLCanvasElement, state: DisplayableState) {
+    let amps = state.amplitudes;
+
+    const measurementHistogram = new Chart(element, {
+        type: 'bar',
+        data: {
+            labels: Array.from(Array(state.n_qubits).keys()).map(idx => {
+                let bitstring = (idx >>> 0).toString(2).padStart(state.n_qubits, "0");
+                return `|${bitstring}⟩`;
+            }), //basis state labels
+            datasets: [
+                {
+                    data: Array.from(Array(state.n_qubits).keys()).map(idx => {
+                        return (amps[idx].Magnitude);
+                    }),
+                    backgroundColor: "#4c4cff",
+                    borderColor: "#4c4cff",
+                    label: "Amplitude"
+                },
+                {
+                    data: Array.from(Array(state.n_qubits).keys()).map(idx => {
+                        return (amps[idx].Phase);
+                    }),
+                    backgroundColor: "#4c4cff",
+                    borderColor: "#4c4cff",
+                    label: "Phase"
+                }
+            ],
+        },
+        options: {
+            responsive: false,
+            legend: {
+                display: false,
+            },
+            scales: {
+                xAxes: [{
+                    scaleLabel: {
+                        display: true,
+                        labelString: 'Basis States'
+                    },
+                    ticks: {
+                        maxRotation: 0,
+                        minRotation: 0
+                    }
+                }],
+                yAxes: [{
+                    scaleLabel: {
+                        display: true,
+                        labelString: 'Meas. Probability'
```

Comment:
Maybe this label string should be something more generic like `'Value'`, which would apply to both the amplitude and phase datasets?